Adding case to ignore T@ in bspwm

LowzG opened the pull request:
<!-- Please read our contributing guide before opening a PR: https://github.com/polybar/polybar/blob/master/CONTRIBUTING.md -->

## What type of PR is this? (check all applicable)

* [ ] Refactor
* [x] Feature
* [ ] Bug Fix
* [ ] Optimization
* [ ] Documentation Update
* [ ] Other: *Replace this with a description of the type of this PR*

## Description
<!--
  Document user-facing changes in this PR (for example: new config options, changed behavior).

  You can also motivate design decisions here.
-->
T@ is producing an warning message in bspwm. Since T@ is not a special state in bspwm I've added a case so that it can be ignored. 
## Related Issues & Documents
<!-- For example: Fixes #1234, Closes #6789 -->
Fixes #2371
## Documentation (check all applicable)

* [ ] This PR requires changes to the Wiki documentation (describe the changes)
* [ ] This PR requires changes to the documentation inside the git repo (please add them to the PR).
* [x] Does not require documentation changes

patrick96 reviewed (changes requested):
@LowzG Welcome and thanks for contributing! The change looks good :)

Could you please also add an entry in the changelog for this fix: https://github.com/polybar/polybar/blob/master/CONTRIBUTING.md#changelog

LowzG commented:
Sorry I missed that @patrick96. I've added an entry to the changelog, also while I was typing mine I noticed the entry above mine was missing a tick so I added it, hope that's ok.

patrick96 reviewed (approved):
Perfect! Thank you very much
